algorithm: mapstitch
slam_preprocessing_enabled: true
known_preprocessing_enabled: true
known_map_config_file: data/known_map.yaml
known_map_frame_id: /known_map_localization/anchor
max_pairwise_distance: 5.0
